lib/scarpe/para.rb

```diff
@@ -10,19 +10,74 @@ def inherited(subclass)
       end
     end
 
+    # Define the display properties for the `Para` widget.
     display_properties :text_items, :stroke, :size, :font, :html_attributes, :hidden
 
+    # Initializes a new instance of the `Para` widget.
+    #
+    # @param args The text content of the paragraph.
+    # @param stroke [String, nil] The color of the text stroke.
+    # @param size [Symbol] The size of the paragraph text.
+    # @param font [String, nil] The font of the paragraph text.
+    # @param hidden [Boolean] Determines if the paragraph is initially hidden.
+    # @param html_attributes [Hash] Additional HTML attributes for the paragraph.
+    # How to Use
+    #
+    # To use the Scarpe::Para class, follow these steps:
+    #
+    # 1. Create a Scarpe::Para instance by calling Scarpe::Para.new and provide the desired text as arguments. Assign it to a variable.
+    # 2. Customize the paragraph by setting optional properties such as stroke, size, font, and hidden.
+    # 3. Replace or modify the paragraph content by calling the replace method and passing new text as arguments.
+    # 4. Hide the paragraph by calling the hide method.
+    # 5. Show the hidden paragraph by calling the show method.
+    #
+    # @example
+    #    Shoes.app do
+    #      para_widget = para "Hello, This is title!", stroke: "red", size: :title, font: "Arial"
+    #      @btn = button "toogle title🪄" ,color:"#FF7116",padding_bottom:"8",padding_top:"8",text_color:"white"
+    #
+    #      toggle = true
+    #
+    #      @btn.click do
+    #        toggle = !toggle
+    #        if toggle
+    #           para_widget.show
+    #        else
+    #           para_widget.hide
+    #        end
+    #       end
+    #
+    #       @btn2 = button "replace title🪄" ,color:"#FF7116",padding_bottom:"8",padding_top:"8",text_color:"white"
+    #
+    #       @btn2.click do
+    #          para_widget.replace("Welcome!")
+    #       end
+    #
+    #       banner_widget = banner("Welcome to Shoes!")
+    #       title_widget = title("Shoes Examples")
+    #       subtitle_widget = subtitle("Explore the Features")
+    #       tagline_widget = tagline("Step into a World of Shoes")
+    #       caption_widget = caption("A GUI Framework for Ruby")
+    #       inscription_widget = inscription("Designed for Easy Development")
+    #
+    #      end
+    #
```

Comment:
hm i am thinking to add image but i cant really figure out relative path and all or what to use i tried using , didnt worked

Reply:
Yeah, may be hard to get a runnable docs example with a local image. Pretty much depends on them having the right image in the right place.

This is also a little verbose -- keep in mind that these are going to be API reference docs, not a full manual. You can *have* manual pages, but they shouldn't be embedded in the widget code.

Reply:
What you want here is more like the manual entry for para, not like how "Nobody Knows Shoes" describes para.